fennel/datasets/test_invalid_dataset.py

```diff
@@ -1150,7 +1150,7 @@ def pipeline1(cls, a: Dataset):
 
     assert (
         str(e.value)
-        == "Window operator over keyed datasets is not defined. Found dataset with keys `['event_id']` in pipeline `pipeline1`"
+        == "Using 'window' param in groupby on keyed dataset is not allowed. Found dataset with keys `['event_id']` in pipeline `pipeline1`."
```

Comment:
Can you add the other invalid tests too, for eg group specifies a different window then what the specific aggregates do, Tumbling forever is not defined, Continuous and discrete cant be mixed together etc

Reply:
added